package.json

```diff
@@ -204,23 +194,19 @@
     "eslint-plugin-prettier": "3.1.2",
     "firebase-admin": "^9.5.0",
     "husky": "^4.2.5",
-    "istanbul-lib-coverage": "^3.0.0",
     "json-schema-to-typescript": "^8.2.0",
     "lint-staged": "^10.1.3",
     "node-fetch": "^2.6.1",
-    "nyc": "^15.0.0",
     "prettier": "2.0.2",
-    "puppeteer": "^3.0.4",
```

Comment:
We use this in a firebase function that generates share images, but that has its own [package.json](https://github.com/covid-projections/covid-projections/blob/cb141f9895a6ad62e0d08bbfba4fc654fc178fce/firebase/functions/package.json#L27).